google-cloud-video_intelligence/.yardopts

```diff
@@ -0,0 +1,8 @@
+--no-private
+--title=Google Cloud Video Intelligence API
+--exclude lib/google/cloud/video_intelligence/v1beta1
```

Comment:
Sounds good. I already added the entry to the main manifest file in the gem setup commit.